m4_sessions/1705-inning/group_six.md

```diff
@@ -10,5 +10,13 @@
 
 ### Outline: 
 
-<p>10 minutes for answering questions. 15 minutes whiteboard process. 25 minutes group discussions. 10 minutes for conclusion. </p>
+  * Answer question on their own (10 minutes):
+     * What is knowledge?
+     
+  * Whiteboard (15 minutes)
+    * Symbolic logic exercise
+  
+  * Group discussions on their answer to question ( 25 minutes )
```

Comment:
What specific questions do you have to prompt the group?  Do you want it to be whole group or small groups of 4?